src/tests/Common/helixpublishwitharcade.proj

```diff
@@ -270,7 +270,7 @@
 
   <ItemGroup Condition=" '$(TestWrapperTargetsWindows)' != 'true' ">
     <HelixPreCommand Include="export CORE_ROOT=$HELIX_CORRELATION_PAYLOAD" />
-    <HelixPreCommand Include="export RunCrossGen=1" Condition=" '$(RunCrossGen)' == 'true' " />
+    <HelixPreCommand Include="export RunCrossGen2=1" Condition=" '$(RunCrossGen)' == 'true' " />
```

Comment:
These seem like weird changes: why not just set `RunCrossGen2=1` in the caller instead?

Reply:
yeah ideally should change the callers, but wanted to keep the changes limited in case we have to revert. Also going forward once we really remove cg1, RunCrossgen==RunCrossgen2, and actually the line below this should be removed.